v2/internal/frontend/desktop/common/process_request.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/wailsapp/wails/v2/internal/frontend/assetserver"
+)
+
+type RequestRespone struct {
+	Body       []byte
+	MimeType   string
+	StatusCode int
+}
+
+func (r RequestRespone) StatusText() string {
+	return http.StatusText(r.StatusCode)
+}
+
+func (r RequestRespone) String() string {
+	return fmt.Sprintf("Body: '%s', StatusCode: %d", string(r.Body), r.StatusCode)
+}
+
+func ProcessRequest(uri string, assets *assetserver.DesktopAssetServer, expectedScheme string, expectedHosts ...string) (RequestRespone, error) {
+	// Translate URI to file
+	file, err := translateUriToFile(uri, expectedScheme, expectedHosts...)
+	if err != nil {
+		if err == ErrUnexpectedHost {
+			body := fmt.Sprintf("expected host one of \"%s\"", strings.Join(expectedHosts, ","))
```

Comment:
```
ERR | Error processing request 'wails://helo.myfonts.net/count/3e7d84': unexpected host (HttpResponse=BODY: 'expected host one of ",null"', StatusCode: 500)
```
Feedback given to the user is a little confusing here?

Reply:
How did you manage to get that output?

Reply:
Any suggestions what you would like to see in the output?

Reply:
Well it just happens! As Lea said on a comment above it just returns null.
I don't know what the intention was originally (show multiple hostnames that one of them if the one user should use?). Looking at it, it is not very clear why there would be multiple hostnames either, so I don't have a suggestion really.
What do you see on Mac/Win? Are we trying to let the user know that some of their assets can not load because they point to the wrong hostname?

Reply:
On MacOS an Windows you see only one suggestion because there's only one hostname that comes through for the assets, which is "wails". Normally users don't have to decide on a hostname in any way, they just have to correctly use relative paths.
Well the two hostnames special case on Linux was already there and I didn't change that. As for that I just output both possibilities, to be fair I'm currently not using Wails on Linux so I have no insights on why there are coming requests with "null" as a hostname.

Reply:
Yeah, I've seen this happen and I'm not sure why. I added code to just "allow" nil hosts and treat it as internal. I think it's fine as is.

Reply:
@leaanthony to reproduce I ran `wails init -n projectname` and added this line to `frontend/src/main.css`
```css
@import url("//helo.myfonts.net/count/3e7d84");
```
which was the original problem. Maybe in the error message hint that probably an incorrect path somewhere in the frontend might be the issue so users have some lead?